examples/preprocessing/plot_run_ica.py

```diff
@@ -25,23 +25,45 @@
 print(__doc__)
 
 ###############################################################################
-# Fit ICA model using the FastICA algorithm, detect and inspect components
+# Read and preprocess the data. Preprocessing consists of:
+# * meg channel selection
```

Comment:
I guess I should add tho spaces at the end of previous line (etc.) for this to render as bullet-point list